fml/platform/android/message_loop_android.cc

```diff
@@ -93,4 +108,33 @@ void MessageLoopAndroid::OnEventFired() {
   }
 }
 
+bool MessageLoopAndroid::Register(JNIEnv* env) {
+  jclass clazz = env->FindClass("android/os/Looper");
+
+  if (clazz == nullptr) {
```

Comment:
simplification nit: this could be a `FML_CHECK`, and the `FML_CHECK` in the consumption side can be dropped.

Reply:
Done